### PR TITLE
(feat) 002: Add unit tests agenda

### PR DESCRIPTION
**Pull request by PrykhodzkaV**

### CODE REVIEW

File: unit-testing/index.md [hunk 1]
```diff
@@ -1,7 +1,20 @@
 # Module content
 
-- ToDo
-- ToDo
-- ToDo
-- ToDo
-- ToDo
+1. Testing in software development​
```
Comment (Liza-Veis) [inline unit-testing/index.md:3]: How about adding an overview of all test types? They are specific in react-native and there are not as many materials on this topic as in unit testing

Comment (PrykhodzkaV) [reply unit-testing/index.md:3]: Hi!
I am gonna cover it briefly via 5. Testing statistic.
for example:
Screenshot 2022-10-05 at 11 07 11

Thanks for your comment :)

Comment (Liza-Veis) [reply unit-testing/index.md:3]: I would also add tools that can be used for this in React Native, and we need to at least add to the materials how to write other types of tests

**Review by Dzmitry-Klokau — APPROVED** [review]

File: unit-testing/index.md [hunk 2]
```diff
@@ -1,7 +1,20 @@
...
+    - testing role
+    - benefits
+    - testing scope
+2. Testing Pyramid
+3. Unit tests
+    - pros and cons
```
Comment (iralitv) [inline unit-testing/index.md:9]: what do you think about a common overview the most popular testing libs for RN?

Comment (DenisKhatsuk) [reply unit-testing/index.md:9]: Agreed. I think it would be nice to highlight the Jest + RN testing library combo.

**Review by annakosakova — APPROVED** [review]

**Pull request merged**